cmd/zetae2e/local/local.go

```diff
@@ -160,11 +160,17 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 
 	zetaTxServer, err := txserver.NewZetaTxServer(
 		conf.RPCs.ZetaCoreRPC,
-		[]string{utils.EmergencyPolicyName, utils.OperationalPolicyName, utils.AdminPolicyName},
+		[]string{
+			utils.EmergencyPolicyName,
+			utils.OperationalPolicyName,
+			utils.AdminPolicyName,
+			utils.UserEmissionsWithdrawName,
```

Comment:
nit: what I did in the distribution precompile e2e tests was creating a different zetaTxServer (with just the precompile user) in the precompile runner config instead of adding users to this zetaTxServer, which is used by every e2e runner.

Not sure what is the desired/best approach here.

Reply:
I think its fine here as the e2e is supposed to be running for every test